common/models/user.js

```diff
@@ -39,14 +36,14 @@ module.exports = function(User) {
   
   // Method to render
   User.afterRemote('prototype.verify', function(context, user, next) {
-    console.log('> user.afterRemote for verify triggered');
-      context.res.render('response', {
-        title: 'A Link to reverify your identity has been sent to your email successfully',
-        content: 'Please check your email and click on the verification link ' +
-            'before logging in',
-        redirectTo: '/',
-        redirectToLinkText: 'Log in'
-      });
+    context.res.render('response', {
+      title: 'A Link to reverify your identity has been sent '+
+      'to your email successfully',
+      content: 'Please check your email and click on the verification link ' +
+      'before logging in',
```

Comment:
We usually indent the second line of a multi-line string by an extra level, to distinguish it from the next property definition.

```js
{
  title: 'A Link to reverify your identity has been sent ' +
    'to your email successfully',
  content: 'Please check your email and click on the verification link ' +
    'before logging in',
}
```